helm/influxdb-athena-crawler/values.yaml

```diff
@@ -24,6 +24,12 @@ defaults:
   # -- After how long to delete the objects.
   maxObjectAge: 5m
 
+  # -- Whether to retain this number of most recent folders
+  retainWindows: ""
```

Comment:
What is the expected format, true/false or is it a duration ?

Reply:
It is a folder count, the whether in the comment was indeed confusing, is it better ?

Reply:
Yes !